tools/benchmark/src/main/java/com/datadog/benchmark/internal/SpanEventSerializer.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.benchmark.internal
+
+import com.datadog.android.core.constraints.DataConstraints
+import com.datadog.android.core.constraints.DatadogDataConstraints
+import com.datadog.android.core.internal.utils.NULL_MAP_VALUE
+import com.datadog.benchmark.internal.model.SpanEvent
+import com.google.gson.JsonArray
+import com.google.gson.JsonObject
+import com.google.gson.JsonPrimitive
+import java.util.Date
+
+internal class SpanEventSerializer(
+    private val dataConstraints: DataConstraints = DatadogDataConstraints()
+) {
+
+    // region Serializer
+
+    fun serialize(env: String, spanEvents: List<SpanEvent>): String {
+        val spans = JsonArray(1)
```

Comment:
why `JsonArray(1)` here ?

Reply:
It seems we only send one span in each `spans` attribute, refer to :
https://github.com/DataDog/dd-sdk-android/blob/develop/features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/internal/domain/event/SpanEventSerializer.kt

Reply:
you are right, now changed to `spanEvents.size`